backend/vkt/src/main/java/fi/oph/vkt/util/LocalisationUtil.java

```diff
@@ -0,0 +1,21 @@
+package fi.oph.vkt.util;
+
+import java.text.MessageFormat;
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+public class LocalisationUtil {
+
+  public static Locale localeFI = Locale.forLanguageTag("fi");
+  public static Locale localeSV = Locale.forLanguageTag("sv");
```

Comment:
Tähän kai vois laittaa `final`. Tai ainakin IDE väittää niin.

```suggestion
  public static final Locale localeFI = Locale.forLanguageTag("fi");
  public static final Locale localeSV = Locale.forLanguageTag("sv");
```

Reply:
Mulla ei IDE mainitse näistä mitään vaikka otin käyttöön tuon sun Readmehen lisäämän asetuksen, joka kyllä muita final määreiden puutteitta havaitseekin. Mut voin lisäillä.

Reply:
Joo, jostain syytä näkyi vaan project errors tabin alla.

<img width="641" alt="Näyttökuva 2023-6-30 kello 13 24 17" src="https://github.com/Opetushallitus/kieli-ja-kaantajatutkinnot/assets/435495/f85cca2f-7ffd-40b4-b29c-f25a8309b881">